aws-redshift-clusterparametergroup/src/main/java/software/amazon/redshift/clusterparametergroup/UpdateHandler.java

```diff
@@ -36,47 +44,110 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         final String resourceName = String.format("arn:%s:redshift:%s:%s:parametergroup:%s", request.getAwsPartition(), request.getRegion(), request.getAwsAccountId(), request.getDesiredResourceState().getParameterGroupName());
 
         return ProgressEvent.progress(request.getDesiredResourceState(), callbackContext)
-                .then(progress ->
-                        proxy.initiate("AWS-Redshift-EventSubscription::Update::ReadTags", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
-                                .translateToServiceRequest(resourceModel -> Translator.translateToReadTagsRequest(resourceName))
-                                .makeServiceCall(this::readTags)
-                                .handleError(this::operateTagsErrorHandler)
-                                .done((tagsRequest, tagsResponse, client, model, context) -> ProgressEvent.<ResourceModel, CallbackContext>builder()
-                                        .callbackContext(callbackContext)
-                                        .callbackDelaySeconds(0)
-                                        .resourceModel(Translator.translateFromReadTagsResponse(tagsResponse))
-                                        .status(OperationStatus.IN_PROGRESS)
-                                        .build()))
+                .then(progress -> proxy.initiate("AWS-Redshift-EventSubscription::Update::ReadTags", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
+                        .translateToServiceRequest(resourceModel -> Translator.translateToReadTagsRequest(resourceName))
+                        .makeServiceCall(this::readTags)
+                        .handleError(this::operateTagsErrorHandler)
+                        .done((tagsRequest, tagsResponse, client, model, context) -> ProgressEvent.<ResourceModel, CallbackContext>builder()
+                                .callbackContext(callbackContext)
+                                .callbackDelaySeconds(0)
+                                .resourceModel(Translator.translateFromReadTagsResponse(tagsResponse))
```

Comment:
what is the diff here? code snippet seems to be the same ?

Reply:
that part has no diff, just clean some whitespaces